cmd/agentctl2/utils/common_utils.go

```diff
@@ -0,0 +1,217 @@
+// Copyright (c) 2017 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"os"
+	"strings"
+
+	"github.com/gogo/protobuf/proto"
+
+	"fmt"
+
+	"github.com/ligato/cn-infra/db/keyval"
+	"github.com/ligato/cn-infra/db/keyval/etcd"
+	"github.com/ligato/cn-infra/db/keyval/kvproto"
+	"github.com/ligato/cn-infra/logging"
+	"github.com/ligato/cn-infra/logging/logrus"
+	"github.com/ligato/cn-infra/servicelabel"
+)
+
+const (
+	ACLPath          = "config/vpp/acls/v2/acl/"
```

Comment:
This shouldn't be redefined. Avaiable like this: https://github.com/ligato/vpp-agent/blob/40bd6b1899e0abe9a30da72c5430225a44cf39b2/plugins/vpp/ifplugin/descriptor/interface.go#L173